pkg/addr/isdas.go

```diff
@@ -213,8 +225,9 @@ func IAFrom(isd ISD, _as AS) (ia IA, err error) {
 }
 
 // ParseIA parses an IA from a string of the format 'isd-as'.
+// @ ensures err != nil ==> err.ErrorMem()
 // @ decreases
-func ParseIA(ia string) (IA, error) {
+func ParseIA(ia string) (ret_ia IA, err error) {
```

Comment:
underscores separating parts of var ids is very unidiomatic in Go/Gobra. Please use camel case

Reply:
I see. Done.